pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -135,6 +148,16 @@ static int _event(struct dtls_context_t *ctx, session_t *session,
     if (!level && (code != DTLS_EVENT_CONNECT)) {
         mbox_put(&sock->mbox, &msg);
     }
+
+    if (IS_ACTIVE(CONFIG_DTLS_ECC) && code == DTLS_EVENT_CONNECTED) {
+        for (unsigned i = 0; i < ARRAY_SIZE(_ecdsa_keys); i++) {
```

Comment:
`_ecdsa_keys` is not declared when `CONFIG_DTLS_ECC` is not set, because `_ecdsa_keys` is declared in a `#ifdef` block. This leads to compilation errors currently.

Reply:
Yep, I'll change it to `ifdef`

Reply:
Done